src/test/regress/expected/multi_colocation_utils.out

```diff
@@ -535,17 +602,51 @@ SELECT create_reference_table('table2_groupF');
 (1 row)
 
 -- check metadata
-SELECT * FROM pg_dist_colocation 
-    WHERE colocationid >= 1 AND colocationid < 1000 
+SELECT * FROM pg_dist_colocation
+    WHERE colocationid >= 1 AND colocationid < 1000
     ORDER BY colocationid;
- colocationid | shardcount | replicationfactor | distributioncolumntype 
```

Comment:
Is it possible to make this outputs more readable? Maybe joining with pg_dist_partition table so that we can also see table names and which table belongs to which colocation group etc.